eth/backend.go

```diff
@@ -490,6 +515,13 @@ func (s *Ethereum) StartMining(threads int) error {
 			log.Error("Cannot start mining without etherbase", "err", err)
 			return fmt.Errorf("etherbase missing: %v", err)
 		}
+
+		val, err := s.Validator()
+		if err != nil {
```

Comment:
Is this necessary? Isn't this what `			log.Info("Validator automatically configured", "address", validator)` does already?